chatlearn/synchronizer/parameter_sync.py

```diff
@@ -265,6 +275,19 @@ def set_send_actors_to_allgather_routed_experts(self, src_replica_ranks_group):
                 self.send_actors_to_allgather_routed_experts[-1].extend(
                     [self.src_model.get_actor(src_rank) for src_rank in src_replica_ranks])
 
+    def set_send_actors_to_alltoall_routed_experts(self, src_replica_ranks_group):
```

Comment:
I think it might be better to integrate this function with `set_send_actors_to_allgather_routed_experts` and rewrite the function signature to `set_send_actors_on_routed_experts(self, src_replica_ranks_group, send_actors_on_routed_experts)`. Then, we can set the parameter `send_actors_on_routed_experts` as `self.send_actors_to_allgather_routed_experts` or `self.send_actors_to_alltoall_routed_experts` when invoking the function from outside.

Reply:
done